### DISCUSSION TITLE
installing bitextor

### DISCUSSION THREAD

**Discussion by iamaazva** [bitextor/bitextor#231] [General]
hi team, I'm new to Github and Bitextor and I am trying to install bitextor on my PC 
could someone please guide me through the installation on Bitextor? I am stuck on this step:
![bitextor installation](https://user-images.githubusercontent.com/103050000/161772416-7dd98204-3dac-4610-b4f0-bffbd358739b.jpeg)

Thank you so much for the help :)

**Comment by cgr71ii** [comment]
Hi!

Could you please share the complete log? I can see that there were cmake errors. Have you checked the [installation guide](https://github.com/bitextor/bitextor/blob/master/docs/INSTALL.md)?
